gravitee-apim-definition/gravitee-apim-definition-model/pom.xml

```diff
@@ -47,6 +47,12 @@
             <groupId>io.gravitee.node</groupId>
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-38820   
**Check Name:** The fix for CVE-2022-22968 made disallowedFieldspatterns in DataBinder ...   
**Severity:** LOW   
**Fixed Version:** 6.1.14, 6.0.25, 5.3.41   
**Reachable Path(s) Found:** No  
**Description:** The fix for CVE-2022-22968 made disallowedFields patterns in DataBinder case insensitive. However, String.toLowerCase() has some Locale dependent exceptions that could potentially result in fields not protected as expected.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-38820